### ISSUE TITLE
pygame.mouse.get_rel() does not take pygame.SCALED flag into account

### ISSUE THREAD

**Issue by derpsticks** [pygame-community/pygame-ce#2022]
Environment:

You can get some of this info from the text that pops up in the console when you run a pygame program.

Operating system (e.g. Windows, Linux(Debian), Linux(Ubuntu), Mac): Windows 1-
Python version (e.g. 3.7.9, 3.8.5) : Python 3.11.0
SDL version (e.g. SDL 2.0.12): SDL 2.0.22
PyGame version (e.g. 2.0.0.dev10, 1.9.6): pygame 2.1.3.dev8
Relevant hardware (e.g. if reporting a bug about a controller, tell us the brand & name of it):
Current behavior:

pygame.mouse.get_rel() is returning a relative offset in native resolution pixels rather than the scaled resolution pixels when using the pygame.SCALED flag.

Expected behavior:

When using the pygame.SCALED flag the get_rel() function should apply the correct scale to the relative mouse position.

**Comment by derpsticks** [comment]
What I am saying is that mouse events are not scaled. If you look at the code related to the get_rel() function you can see that the part where scaling is applied has been commented out:

[pygame/src_c/mouse.c](https://github.com/pygame/pygame/blob/f86fa03541d6e4b5a8523f1947adc1b4f826d797/src_c/mouse.c#L104)

Line 104 in [f86fa03](https://github.com/pygame/pygame/commit/f86fa03541d6e4b5a8523f1947adc1b4f826d797)

 mouse_get_rel(PyObject *self, PyObject *_null) 
Right now the code is just returning the result of SDL_GetRelativeMouseState without any scaling applied. Unless the scaling is applied later to the returned tuple then I'm not sure there is any scaling at all.

**Comment by robertpfeiffer** [comment]
This was left this way somewhat deliberately, because the other way round, sensitivity in FPS games would depend on window size

**Comment by robertpfeiffer** [comment]
I would like to close this as not a bug (I mean it), but I might be persuaded to change this behaviour depending on whether the mouse pointer is shown and moving around or captured (in FPS games).

**Comment by bigwhoopgames** [comment]
I think that if the developer is using the SCALED mode and the documentation literally says "Mouse events are scaled for you, so your game doesn't need to do it." then perhaps the function should reflect that and scale the mouse events for you. Perhaps add another get_rel_absolute() function that returns the relative mouse position not scaled or add a kwarg to toggle if the return is scaled or not (default being return scaled relative pos if SCALED enabled).

**Comment by bigwhoopgames** [comment]
For what it is worth this issue was closed in upstream pygame: https://github.com/pygame/pygame/issues/3585 so further changes will results in more drift between pygame and pygame-ce

**Comment by robertpfeiffer** [comment]
Yeah, he un-commented something I wrote. If you really want this, I would try to add it back in, but guard it behind
SDL_ShowCursor(SDL_QUERY), so the mouse sensitivity still works normally when the mouse is captured. And then we'd need to double check how this differs from events, which AFAIK are or were already scaled by SDL itself.
